woodwork/schema.py

```diff
@@ -327,6 +327,43 @@ def set_time_index(self, new_time_index):
             _check_time_index(self.columns.keys(), new_time_index, self.logical_types.get(new_time_index))
             self._set_time_index_tags(new_time_index)
 
+    def rename(self, columns):
+        """Renames columns in a Schema
+
+        Args:
+            columns (dict[str -> str]): A dictionary mapping columns whose names
+                we'd like to change to the name to which we'd like to change them.
```

Comment:
Maybe we can simplify this a bit?  Perhaps something like:
```
columns (dict[str -> str]): A dictionary mapping current column names to new column names.
```

Reply:
This is much cleaner! Changing